rules/linux/privilege_escalation_enlightenment_window_manager.toml

```diff
@@ -0,0 +1,73 @@
+[metadata]
+creation_date = "2024/01/05"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
```

Comment:
I think we should provide a min stack comment here, for the new process capability fields

Reply:
Would providing the ECS files make sense or just a description like "supports process capabilities [ effective, permitted ]

Reply:
I'd suggest something along the line of "Linux process capability data sources were added in version 8.11.0".